static/app/utils/prioritySort.tsx

```diff
@@ -0,0 +1,40 @@
+import ConfigStore from 'sentry/stores/configStore';
+import {Organization} from 'sentry/types';
+
+export function prioritySortExperimentEnabled(organization: Organization) {
+  /**
+   * Return true if the experiment is enabled for this organization and that we should log the experiment
+   */
+  return (
+    organization.isEarlyAdopter &&
```

Comment:
Hmm is this check really necessary? Since we already define the feature flags

Reply:
@malwilley it's to make it idiot proof lmfao